semantic/src/site/elements/label.overrides

```diff
@@ -22,7 +27,26 @@
   background-color: transparent;
   color: @darkGrey;
   font-weight: @regularWeight;
-  font-size: @fontXXsmall;
   height: 20px;
   padding-left: 0;
 }
+
+.ui.label.text-italic {
+  font-style: italic;
+}
+
+.ui.label.simple-label > .text-blue {
```

Comment:
Wouldn't we just make a generic "info-text" class that can be used for ALL text elements that need to be coloured `@information` colour? No need to restrict it to one particular label type.

Reply:
Sure. Good idea. I'll move to the `site.overrides` file